rero_ils/theme/templates/rero_ils/page_settings.html

```diff
@@ -20,43 +20,21 @@
 {%- extends config.THEME_BASE_TEMPLATE %}
 
 {%- block body scoped %}
-  <div class="row">
-    <div class="col-md-2 col-pad-left-0">
-      {%- block settings_menu scoped %}
-      <ul class="list-group">
-        {%- for item in current_menu.submenu('settings').children if item.visible %}
-        {%- block settings_menu_item scoped %}
-        <a href="{{ item.url }}" class="list-group-item{% if item.active %} active{% endif %}">
-          {{ item.text|safe }}
-        </a>
+  {%- block settings_content scoped %}
+  <div class="panel-group panel-bot-margin mt-2">
```

Comment:
`panel-bot-margin mt-2` seems wired. Why two margins?

Reply:
I deleted the class `panel-bot-margin`